src/contexts/TodoContext.tsx

```diff
@@ -0,0 +1,68 @@
+import { useReducer, createContext } from "react";
+import { IState, IAction } from "interface";
+
+// localStorage에서 todoList, doneList item 불러오기
+const todoStorage = localStorage.getItem("todoList");
+const doneStorage = localStorage.getItem("doneList");
+
+let loadTodo = todoStorage ? JSON.parse(todoStorage) : [];
+let loadDone = doneStorage ? JSON.parse(doneStorage) : [];
+
+// 전역 데이터의 초기값
+const initialState = {
+  todoList: loadTodo,
+  doneList: loadDone,
+  dispatch: () => {},
+};
```

Comment:
```suggestion
const initialState: IState = {
  todoList: loadTodo,
  doneList: loadDone,
  dispatch: () => {},
};
```
상태는 타입을 꼭 명시해주세요...^^

Reply:
아이코 저걸 빼먹었네요.. 감사합니다!!